compliance_suite/test_runner.py

```diff
@@ -261,14 +262,14 @@ def run_tests(
             if "env_vars" in self.job_data.keys() and "check_cancel" in self.job_data["env_vars"].keys():
                 check_cancel = self.job_data["env_vars"]["check_cancel"]
 
-            response = client.poll_request(service=self.service, server=self.server, version=self.version,
+            response = client.poll_request(service=self.service, server=self.server, version=self.version[0:2],
```

Comment:
This puts quite some assumptions on how the version looks like. We currently deal with `v1.0` and `v1.1` in TES, but note that including `v` is non-standard according to [SemVer](http://semver.org/). It's also possible to have major, minor or patch versions with more than one digit.

I think you should better write/find/import a SemVer parser (that may optionally have a leading `v`).